src/Logging/InitializationLogger.cs

```diff
@@ -41,7 +41,7 @@ private static ILoggingBuilder LoadInitializationLogger(this ILoggingBuilder bui
 		public static void LogInitializationSucceed(string serviceNameForLogging, string message = "")
 		{
 			string logMessage = $"Initialization successful for {serviceNameForLogging}, {message}";
-			ServiceInitializationEventSource.Instance.LogHostBuildSucceeded(Process.GetCurrentProcess().Id, serviceNameForLogging, logMessage);
+			ServiceInitializationEventSource.Instance.LogHostBuildSucceeded(Environment.ProcessId, serviceNameForLogging, logMessage);
```

Comment:
Is Environment.ProcessId correct? I see some failures below saying there is no definition for 'ProcessId' ?

Reply:
Yes, as per my comment above Environment.ProcessId doesn't exist in NetStandard 2.0.
It was only added in Net5. (https://docs.microsoft.com/en-us/dotnet/api/system.environment.processid?view=net-6.0)
The main benefits are that it:
- Caches the ID for any future calls.
- Prevents creation of a new process, just to get the ID.
- Avoids memory leaks due to developers forgetting to dispose of the Process instance that is created by GetCurrentProcess()

To get around the Net5+ requirement, I've specified conditional inclusion of this logic, and wrapped the NetStandard code in a using, to ensure the process gets disposed.

Reply:
Changed the variables used based on the doc Andrey linked.
I think the use of NET5_0_OR_GREATER adds more context.